src/components/rectificationListRow/RectificationListRow.tsx

```diff
@@ -40,7 +40,7 @@ const RectificationListRow: FC<Props> = ({ form }): React.ReactElement => {
   /* Gets foul/transfer data and possible decision from PASI to be shown
   with the objection form. If data can't be fetched from PASI for some
   reason, the objection form can still be shown */
-  const fetchDataAndSetExtended = async () => {
+  const fetchData = async () => {
```

Comment:
This function still does set the extended status, so I would not rename it into fetchData without splitting the function into 2 separate ones.

If I were to refactor this I would approach this with something like handleExtend function which calls functions setExtended and and fetchData.